e2e/openOsmWithUser.spec.ts

```diff
@@ -8,27 +8,42 @@ let projects = getProjects();
 projects = projects.filter((arg) => arg.name !== 'atlas');
 
 for (const project of projects) {
-  test(`As User with no rights, I can go to ${project.title}, open map and open Rapid OSM editor at map coordinates`, async ({
-    context,
-    pageManager,
-  }) => {
-    await pageManager.atBrowser.openProject(project, { skipCookieBanner: true });
-    await pageManager.fromNavigationMenu.goToMap();
-
-    // TO DO: remove this action after 18582 issue is fixed
-    await pageManager.atMap.goToSpecificAreaByUrl(10.597, 53.9196, 27.5097, project);
-
-    const coordinates = await pageManager.atMap.getViewportFromUrl();
-
-    const [newPage] = await Promise.all([
-      context.waitForEvent('page'),
-      (await pageManager.atToolBar.getButtonByText('Edit map in OSM')).click({
-        delay: 330,
-      }),
-    ]);
-
-    await pageManager.atMap.waitForUrlToMatchPattern(/rapideditor/, newPage);
-    const osmCoordinates = await pageManager.atMap.getViewportFromUrl(newPage);
-    expect(osmCoordinates).toStrictEqual(coordinates);
-  });
+  const editors: [string, RegExp][] = [
+    ['OpenStreetMap.org default editor', /openstreetmap/],
+    ['RapiD', /rapideditor/],
+    ['iD', /openstreetmap.*editor%3Did/],
+    // ['JOSM', /load_and_zoom/], // TODO: Fix that waitForUrlToMatchPattern is not working with JOSM
```

Comment:
not sure that running JOSM on VM where we run tests is at all possible. Playwright I suppose does not support testing desktop apps

Reply:
so you can delete this comment, or you see the way to test it in automatic way?

Reply:
I just wanted to check that the josm link opens in a new tab. In other tests it works similarly, but in this case I suspect that it may not work because this resource is unavailable